apps/design-system-docs/docs/components/packages/wp-components/status-indicator.tsx

```diff
@@ -0,0 +1,16 @@
+import { Tooltip } from '@wordpress/components';
+import { ComponentStatus, statuses } from './data';
+import styles from './status-indicator.module.scss';
+
+export const StatusIndicator = ( { status }: { status: ComponentStatus } ) => {
+	return (
+		<Tooltip text={ statuses.find( ( s ) => s.value === status )?.label }>
```

Comment:
Since we're using the corresponding `statuses` value 3 times, maybe we ought to assign a variable?

```suggestion
	const { label, icon } = statuses.find( ( s ) => s.value === status )!;

	return (
		<Tooltip text={ label }>
```

Reply:
Good catch, fixed in 998308b.